zeppelin-web/src/app/notebook/paragraph/paragraph-chart-selector.html

```diff
@@ -13,49 +13,55 @@
 -->
 
 <div id="{{paragraph.id}}_switch"
-     ng-if="(paragraph.result.type == 'TABLE' || apps.length > 0 || suggestion.available && suggestion.available.length > 0) && !asIframe && !viewOnly"
+     ng-if="(paragraph.result.type == 'TABLE' || paragraph.result.type == 'NETWORK' || apps.length > 0 || suggestion.available && suggestion.available.length > 0) && !asIframe && !viewOnly"
      class="btn-group"
      style="margin-bottom: 10px;">
 
   <button type="button" class="btn btn-default btn-sm"
-          ng-if="paragraph.result.type == 'TABLE'"
+          ng-if="paragraph.result.type == 'TABLE' || paragraph.result.type == 'NETWORK'"
```

Comment:
Cool! could you add examples/info on this to [DisplaySystem](http://zeppelin.apache.org/docs/latest/displaysystem/basicdisplaysystem.html)?